### PR TITLE
diesel 2.0 support

### LINKED ISSUE #2: Update to diesel 2.0
When changing the version to 2.0, the build fails with

```rust
> rustup run stable cargo build
    Updating crates.io index
   Compiling version_check v0.9.4
   Compiling syn v1.0.107
   Compiling serde v1.0.152
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling serde_derive v1.0.152
   Compiling diesel_derives v2.0.1
   Compiling serde_json v1.0.91
   Compiling diesel v2.0.3
   Compiling diesel_json v0.1.1 (/home/jayvdb/rust/diesel_json)
warning: #[sql_type] attribute form is deprecated
  = help: use `#[diesel(sql_type = sql_types::Jsonb)]` instead

warning: #[sql_type] attribute form is deprecated
  = help: use `#[diesel(sql_type = sql_types::Jsonb)]` instead

error[E0107]: this struct takes 1 generic argument but 2 generic arguments were supplied
  --> src/lib.rs:84:38
   |
84 |         out: &mut diesel::serialize::Output<W, Pg>,
   |                                      ^^^^^^    -- help: remove this generic argument
   |                                      |
   |                                      expected 1 generic argument
   |
note: struct defined here, with 1 generic parameter: `DB`
  --> /home/jayvdb/.cargo/registry/src/github.com-1ecc6299db9ec823/diesel-2.0.3/src/serialize.rs:35:12
   |
35 | pub struct Output<'a, 'b, DB>
   |            ^^^^^^         --

For more information about this error, try `rustc --explain E0107`.
```

### PR DESCRIPTION
**Pull request by jayvdb**
Closes https://github.com/PPakalns/diesel_json/issues/2

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -79,12 +81,11 @@ impl<T> ToSql<sql_types::Jsonb, Pg> for Json<T>
...
-        &self,
-        out: &mut diesel::serialize::Output<W, Pg>,
-    ) -> diesel::serialize::Result {
-        let value = serde_json::to_value(self)?;
-        <serde_json::Value as ToSql<sql_types::Jsonb, Pg>>::to_sql(&value, out)
+    fn to_sql(&self, out: &mut diesel::serialize::Output<Pg>) -> diesel::serialize::Result {
+        out.write_all(&[1])?;
+        serde_json::to_writer(out, self)
+            .map(|_| IsNull::No)
+            .map_err(Into::into)
```
Comment (popen2) [inline src/lib.rs]: Sorry to barge in, I was just looking to see if somebody had already created a diesel-2 fork 😀 Thank you for that!
```suggestion
        let value = serde_json::to_value(self)?;
        <serde_json::Value as ToSql<sql_types::Jsonb, Pg>>::to_sql(&value, &mut out.reborrow())
```

Comment (jayvdb) [reply src/lib.rs]: Thanks for that suggestion!

**Comment by PPakalns** [comment]
Hello, thank you for providing changes to support new diesel version.

Will test these changes and release crate under new breaking changes version.

**Comment by PPakalns** [comment]
New version with necessary changes released:
diesel_json = "0.2.0" 

Thank you!

**Pull request merged**